### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by Sluce714** [xJon/The-1.12.2-Pack#847]
**Basic details**
1. Are you using a legitimate launcher and Minecraft account:
 yes
2. Are you using a computer with 8GB or more of RAM? ([How to check](https://www.wikihow.com/Check-Computer-RAM)):
 yes
3. Are you using Java 8, 64-bit? ([How to check](https://www.wikihow.com/Determine-Java-Version)):
yes

**Information required**
1. Is the modpack crashing: yes
2. Crash/latest log *link* ([Where to find](https://github.com/xJon/The-1.12.2-Pack/blob/master/log_location.md#finding-modpack-installation-folder)): **link to paste**
3. Is Optifine installed or any other additional mods: no

**Describe the issue**

A clear and concise description of what the issue is.
After I run the progress bar
Pressing any option will jump back to Technic Launcher
Already reinstalled the mod pack again 
just stay here
Otherwise you have to restart

![image](https://user-images.githubusercontent.com/95038840/208230742-2da16d28-4e58-4786-9953-947f56acaf07.png)
(Sorry, English is not my country's language
I had to use google translate
you may have difficulty seeing)
[新文字文件 (4).txt](https://github.com/xJon/The-1.12.2-Pack/files/10251160/4.txt)

**Comment by xJon** [comment]
So the modpack launches but once you try to enter a world you crash?

**Comment by Sluce714** [comment]
Yes when I want to choose any option will crash

Yesterday was the first day of playing, and it was still possible to enter normally
Can't play today

**Comment by xJon** [comment]
Can you include a crash log from the `crash-reports` folder or are there none?

**Comment by Sluce714** [comment]
there none

**Comment by Sluce714** [comment]
![image](https://user-images.githubusercontent.com/95038840/208232664-67fdff00-61ae-4ae8-8b51-6aceba10d355.png)

**Comment by xJon** [comment]
Did you change any default settings in-game (e.g. modify mipmap settings)?

**Comment by Sluce714** [comment]
no

**Comment by xJon** [comment]
So it keeps just exiting the game, and in the log does it always mention `zone.rong.loliasm.client.sprite.FramesTextureData` as the last thing? Can you check?
Disabling `releaseSpriteFramesCache` in `\config\loliasm.cfg` might help.

**Comment by Sluce714** [comment]
I've tried many times and still can't do it.
When I switch minecraft to other screens (ex:discord, chrome) will exit the game

**Comment by Sluce714** [comment]
wait,I seem to have changed the default settings in game

**Comment by Sluce714** [comment]
You mean it can't be changed like this?
![image](https://user-images.githubusercontent.com/95038840/208240102-9348cf93-15db-4d43-a89a-808cf61f0390.png)

**Comment by xJon** [comment]
I don't have any knowledge of those particular settings affecting the game (as long as `Mipmap Levels` is `OFF`).
Could you attach another log, from a session right after it crashed?

**Comment by Sluce714** [comment]
[latest.log](https://github.com/xJon/The-1.12.2-Pack/files/10258001/latest.log)

**Comment by xJon** [comment]
@Sluce714 Is that log from a session in which the game closed? In the log it seems like it loaded the world fine.

**Comment by Sluce714** [comment]
Yes he turned on successfully but bounced back anyway.  But I restarted it again and it worked, which is a pleasant surprise

**Comment by xJon** [comment]
So it does work? In any case I'd need a log from the exact session it does crash, so I can find the relevant error :)

**Comment by xJon** [comment]
Any update on this?
